### DISCUSSION TITLE
微调后找不到bin文件

### DISCUSSION THREAD

**Discussion by Lurenjia-LRJ** [zai-org/ChatGLM3#512] [Q&A]
Model weights saved in output/advertise_gen_pt-20231203-101222-128-2e-2\pytorch_model.bin
日志显示保存到这个文件下，但是没有
![捕获](https://github.com/THUDM/ChatGLM3/assets/145238822/c366729c-49dd-477e-8059-e8272656a7f1)

**Comment by zRzRzRzRzRzRzR** [comment]
#253 在这里讨论
